package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix",
     "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc",
-    "swagger": "node ./swagger.js"
+    "swagger": "node ./swagger/swagger.js"
```

Comment:
I updated the relative path to the Swagger generator file since there is a dedicated swagger file at the root of the project.